Move docs (#4457)
is-a-dev/register: docs/github/index.md

> # Setting up GitHub pages with is-a.dev subdomain

Reviewer: ```suggestion
# GitHub Pages
```

> ## Creating a GitHub pages repository

Reviewer: ```suggestion
## 1. Creating a repository
```

> You can create a GitHub pages website by creating a repository with the name `<github-username>.github.io`. For more information about GitHub pages, please read through [their guide](https://guides.github.com/features/pages).

Reviewer: ```suggestion
**If you have already created a repository and setup GitHub Pages, you can skip this step.

You can create a GitHub Pages website by creating a repository with the name `<github-username>.github.io`. For more information about GitHub Pages, please read through the GitHub Pages [docs](https://guides.github.com/features/pages).
```

> ## Creating the domain file

Reviewer: ```suggestion
## 2. Register a subdomain
```

> Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content

Reviewer: ```suggestion
First, create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content:

```

```json 
{

>     "description": "Describe the use of this subdomain",

Reviewer: ```suggestion
    "description": "A description of your website",
```

    "repo": "https://github.com/<github-username>/<github-username>.github.io",
    "owner": {
        "username": "<github-username>",

>         "email": "email@address",

Reviewer: ```suggestion
        "email": "email@address.tld"
```

>         "twitter": "<twitter-username>"

Reviewer: ```suggestion
```

Remove as it is not a requirement, only email is.

    },
    "record": {

>         "CNAME": "<username>.github.io"

Reviewer: ```suggestion
        "CNAME": "<github-username>.github.io"
```

    }
} 
```

> ## Configuring

Reviewer: ```suggestion
## 3. Configure your subdomain
```

> - After the pull request is merged, you will see a **404** error on `<your-subdomain>.is-a.dev`. To fix this, go to your GitHub pages repository's **Settings > GitHub pages > Custom domain** and add `<your-subdomain>.is-a.dev` in the given field. _Only do this **after** your pull request is merged._

Reviewer: ```suggestion
- **After your pull request is merged**, you will see a **404** error on `<subdomain>.is-a.dev`. To fix this, go to your repository and go to: **Settings > GPages > Custom Domain** and add `<subdomain>.is-a.dev` in the given field. __Only do this **after** your pull request is merged.__
```

> - Check the **Enforce HTTPS** checkbox below the custom domain input.

Reviewer: ```suggestion
- Check the **Enforce HTTPS** checkbox below the custom domain input, after the SSL certificate has been generated.
```
